actions/websocket_actions.jsx

```diff
@@ -881,17 +881,15 @@ function handleChannelViewedEvent(msg) {
     }
 }
 
-function handlePluginEnabled(msg) {
+export function handlePluginEnabled(msg) {
```

Comment:
not a huge fan of exposing these for testing, proposed a better refactor solution for later and filed a jira for it: https://mattermost.atlassian.net/browse/MM-17894